.github/workflows/build_debug_apk.yml

```
@@ -141,7 +141,7 @@ jobs:
           name: Debug APK
           path: artifacts/debug-apk
       - name: Export APK_NAME for later use
-        run: echo "APK_NAME=Lawnicons.Debug.${{ github.ref_name }}.Nightly-CI_${{ github.run_number }}-$(echo ${{ github.sha }} | cut -c1-7).apk" >> $GITHUB_ENV
+        run: echo "APK_NAME=Lawnicons.Nightly(${{ github.ref_name }}-$(echo ${{ github.sha }} | cut -c1-7)).apk" >> $GITHUB_ENV
```

We could probably simplify this further and just show the commit hash:


```suggestion
        run: echo "APK_NAME=Lawnicons.Nightly.$(echo ${{ github.sha }} | cut -c1-7).apk" >> $GITHUB_ENV
```